### PR TITLE
frostwire: 6.6.7 -> 6.9.3

### PR DESCRIPTION
**Pull request by lorenzleutgeb**
###### Things done

- [ ] Tested using sandboxing ([nix.useSandbox](https://nixos.org/nixos/manual/options.html#opt-nix.useSandbox) on NixOS, or option `sandbox` in [`nix.conf`](https://nixos.org/nix/manual/#sec-conf-file) on non-NixOS linux)
- Built on platform(s)
   - [ ] NixOS
   - [ ] macOS
   - [ ] other Linux distributions
- [ ] Tested via one or more NixOS test(s) if existing and applicable for the change (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
- [ ] Tested compilation of all pkgs that depend on this change using `nix-shell -p nixpkgs-review --run "nixpkgs-review wip"`
- [ ] Tested execution of all binary files (usually in `./result/bin/`)
- [ ] Determined the impact on package closure size (by running `nix path-info -S` before and after)
- [ ] Ensured that relevant documentation is up to date
- [ ] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/.github/CONTRIBUTING.md).

### CODE REVIEW

File: pkgs/applications/networking/p2p/frostwire/default.nix [hunk 1]
```diff
@@ -1,14 +1,14 @@
 { lib, stdenv, fetchFromGitHub, gradle, perl, jre, makeWrapper, makeDesktopItem, mplayer }:
 
 let
-  version = "6.6.7-build-529";
+  version = "6.9.3-build-305";
   name = "frostwire-desktop-${version}";
```
Comment (SuperSandro2000) [inline pkgs/applications/networking/p2p/frostwire/default.nix:5]:
```suggestion
  pname = "frostwire-desktop";
  version = "6.9.3-build-305";
```

**Comment by r-rmcgibbo** [comment]
Result of `nixpkgs-review pr 119375` at fc84b559 run on x86_64-linux [1](https://github.com/Mic92/nixpkgs-review)
<details>
  <summary>1 package failed to build:</summary>
  <ul>
    <li><a href="https://gist.github.com/6ba1386fea91cd7c3e4678a6ce5370f0">frostwire</a></li>
  </ul>
</details>
<details>
  <summary>3 suggestions:</summary>
  <ul>
    <li>warning: <a href="https://github.com/jtojnar/nixpkgs-hammering/blob/master/explanations/missing-phase-hooks.md">missing-phase-hooks</a>

`installPhase` should probably contain `runHook preInstall` and `runHook postInstall`.

Near pkgs/applications/networking/p2p/frostwire/default.nix:70:3:
```
   |
70 |   installPhase = ''
   |   ^
```
</li>
    <li>warning: <a href="https://github.com/jtojnar/nixpkgs-hammering/blob/master/explanations/unclear-gpl.md">unclear-gpl</a>

`gpl2` is a deprecated license, please check if project uses `gpl2Plus` or `gpl2Only` and change `meta.license` accordingly.

Near pkgs/applications/networking/p2p/frostwire/default.nix:90:5:
```
   |
90 |     license = licenses.gpl2;
   |     ^
```
</li>
    <li>warning: <a href="https://github.com/jtojnar/nixpkgs-hammering/blob/master/explanations/missing-phase-hooks.md">missing-phase-hooks</a>

`buildPhase` should probably contain `runHook preBuild` and `runHook postBuild`.

Near pkgs/applications/networking/p2p/frostwire/default.nix:52:3:
```
   |
52 |   buildPhase = ''
   |   ^
```
</li>
  </ul>
</details>

Note that build failures [may predate](https://github.com/nix-community/hydra-check) this PR, and could be nondeterministic or hardware dependent.
Please exercise your independent judgement.

**Comment by SuperSandro2000** [comment]
@ofborg build frostwire

**Comment by SuperSandro2000** [comment]
fails to build with 

```
configuring
no configure script, doing nothing
building
Welcome to Gradle 6.8.3!
Here are the highlights of this release:
 - Faster Kotlin DSL script compilation
 - Vendor selection for Java toolchains
 - Convenient execution of tasks in composite builds
 - Consistent dependency resolution
For more details see https://docs.gradle.org/6.8.3/release-notes.html
> Task :compileJava FAILED
FAILURE: Build failed with an exception.
* What went wrong:
Execution failed for task ':compileJava'.
> Could not target platform: 'Java SE 11' using tool chain: 'JDK 8 (1.8)'.
* Try:
Run with --stacktrace option to get the stack trace. Run with --info or --debug option to get more log output. Run with --scan to get full insights.
* Get more help at https://help.gradle.org
BUILD FAILED in 8s
1 actionable task: 1 executed
builder for '/nix/store/c0f5qv3z6z0840j2gvmsg51x3qypb047-frostwire-desktop-6.9.3-build-305-deps.drv' failed with exit code 1
cannot build derivation '/nix/store/msjx1crpl1h8hnp2j584840zick4z0ld-frostwire-desktop-6.9.3-build-305.drv': 1 dependencies couldn't be built
error: build of '/nix/store/msjx1crpl1h8hnp2j584840zick4z0ld-frostwire-desktop-6.9.3-build-305.drv' failed
```

**Pull request closed without merging**